documentation/content/en/chapters/admin_guide.xml

```diff
@@ -2487,9 +2487,9 @@
 				</authorgroup>
 			</sectioninfo>
 			<title>Checking for updates</title>
-			<para>The phpBB 3.3.x branch is usually updated every couple of months as necessary. Bugfixes, new features and other changes are included in these updates. The minor version number gets incremented each time. It is strongly recommended to keep your phpBB installation up to date. Updating from older versions is more difficult and you will have a hard time finding solutions to possible conflicts. You can update with the Automatic Update Package, which is able to merge modifications from MODs with the updates or you can use one of the other packages provided.</para>
+			<para>The phpBB 3.3.x branch is usually updated every couple of months as necessary. Bugfixes, new features and other changes are included in these updates. The minor version number gets incremented each time. It is strongly recommended keeping your phpBB installation up to date. Updating from older versions is more difficult, and you will have a hard time finding solutions to possible conflicts. You should use the Full Package Update to update your board, especially if your board does not have any modifications to core files. Alternatively expert users may want to update with the Advanced Update Package, which is able to merge modifications to core files.</para>
 			<para>You will be notified in your ACP if a new version is released, you will also have a link to the newest release announcement, which will brief you on the added features and the overall changelog.</para>
-			<para>Updating with the Automatic Update Package is very simple. First, you will go to the linked phpBB.com downloads page and download the appropriate file. You will extract the contents on your PC and upload them to the root directory of your board. The board will be offline for normal users for the moment. Then simply go to the <filename class='directory'>install/</filename> directory and select the <guilabel>Update</guilabel> tab, the updater will then give you further instructions.</para>
+			<para>Updating with the Full Package is very simple. First, you will go to the linked phpBB.com downloads page and download the full package of the latest version. You will extract the contents on your PC and upload them to the root directory of your board, overwriting any old files. The board will be offline for normal users for the moment. Then simply go to the <filename class='directory'>install/</filename> directory and select the <guilabel>Update</guilabel> tab, continue with updating your board by clicking on  "Update database only".</para>
```

Comment:
> You will extract the contents on your PC and upload them to the root directory of your board, overwriting any old files.

This will break your board because of the empty config.php file and depending on your OS might clear avatar & attachments uploads and store directory. Maybe you can link to [these instructions](https://www.phpbb.com/support/docs/en/3.2/ug/upgradeguide/upgrade31/) or perhaps create a new package that don't include these folders.

Reply:
Added a link.